package.json

```diff
@@ -18,14 +19,15 @@
     "@types/react-dom": "^16.9.8",
     "date-fns": "^2.16.1",
     "fhirpath": "^2.9.0",
-    "fqm-execution": "^0.3.1",
+    "fqm-execution": "^0.4.0",
```

Comment:
In theory you could update this to `^0.5.0` now, if you wanted.

Reply:
oops! It actually is locally, just didn't push the changes.